agent/security_helper.py

```diff
@@ -249,3 +250,17 @@ def audit_sshd():
         if insecure_params[param] != config[param][1]:
             issues[param] = insecure_params[param]
     return issues
+
+
+def mysql_root_access():
+    try:
+        subprocess.check_call(["mysql", "-uroot", "-eSHOW DATABASES;"], timeout=5,
+                              stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        return True
```

Comment:
pls move this out of the `try` clause

Reply:
Why? What's going to change?

Reply:
In this particular case - nothing besides readability. But in general a good practice is to not put in `try` anything besides the code supposed to throw an exception in some condition

Reply:
Splitting these two lines into two blocks, one of which will be at the bottom, will decrease readability. For no reason whatsoever.

Reply:
I don't agree, but let's forget about subjective readability. What about following good practice rules?